idxfile.php.dist

```diff
@@ -145,4 +175,5 @@ $idx->
 
 $idx->addLibrary('deploy', new Deploy());
 $idx->addLibrary('phpunit', new PHPUnit());
+$idx->addLibrary('slack', new Slack($slackParams));
```

Comment:
Is this the way we want do to this? Maybe is Idephix responsibility to inject parameters in the extension, a la Symfony2, 
What do you think guys?